spec/unit/vim_plug_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'exogenesis/passengers/vim_plug'
+
+describe VimPlug do
+  let(:config) { double }
+  let(:executor) { executor_double }
+
+  subject { VimPlug.new(config, executor) }
+
+  describe :up do
+    it 'should install and update plugins' do
+      expect(executor).to receive(:execute_interactive).with('Installing and Updating Vim Bundles', 'vim +PlugInstall\! +qall')
+      subject.up
+    end
+  end
+
+  describe :clean do
+    it 'should interactively execute BundleClean' do
+      executor.should_receive(:execute_interactive).with('Cleaning', 'vim +PlugClean\! +qall')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.